RELEASE-NOTES.txt

```diff
@@ -3,10 +3,13 @@
 * [*] [Jetpack-only] Reader: Fix the display of the followed topics in the filter sheet when they're written in non-latin languages. [#20702]
 * [*] [Jetpack-only] Reader: Fixed an issue that was causing the Report and Block actions to be missing from Post Menu actions sheet. [#20705]
 * [***] Enables editing of the site homepage for sites using block-based themes directly from the pages list. [#20713]
+* [*] Add disabled style to `Cell` component [https://github.com/WordPress/gutenberg/pull/50665]
+* [**] Fix undo/redo history when inserting a link configured to open in a new tab [https://github.com/WordPress/gutenberg/pull/50460]
+* [**] Disable details settings for not belonged VideoPress videos [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782]
+* [*] [List block] Fix an issue when merging a list item into a Paragraph would remove its nested list items [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5785]
```

Comment:
Usually, we add the prefix `Block Editor: ` to the changes coming from the editor ([example](https://github.com/wordpress-mobile/WordPress-iOS/blob/e4a7849fc5501d83aa55ad46f9f2e7d4c11f669e/RELEASE-NOTES.txt#LL17C10-L17C10)).